src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/AddNewUserTest.java

```diff
@@ -16,15 +19,43 @@
 @ActiveProfiles("functional")
 public class AddNewUserTest extends AuthorizationFunctionalTest {
 
+    String orgIdentifierResponse = null;
+
+    @Before
+    public void createAndUpdateOrganisation() {
+        Map<String, Object> response = professionalApiClient.createOrganisation();
+        orgIdentifierResponse = (String) response.get("organisationIdentifier");
+        assertThat(orgIdentifierResponse).isNotEmpty();
+        professionalApiClient.updateOrganisation(orgIdentifierResponse, hmctsAdmin);
+    }
 
     @Test
     public void add_new_user_to_organisation() {
 
-        String organisationIdentifier = createAndUpdateOrganisationToActive(hmctsAdmin);
+        NewUserCreationRequest newUserCreationRequest = professionalApiClient.createNewUserRequest();
+        Map<String, Object> newUserResponse = professionalApiClient.addNewUserToAnOrganisation(orgIdentifierResponse, hmctsAdmin,  newUserCreationRequest, HttpStatus.CREATED);
+        assertThat(newUserResponse).isNotNull();
+    }
+
+    @Test
+    public void add_new_user_to_organisation_with_no_jurisdiction() {
 
         NewUserCreationRequest newUserCreationRequest = professionalApiClient.createNewUserRequest();
-        Map<String, Object> newUserResponse = professionalApiClient.addNewUserToAnOrganisation(organisationIdentifier, hmctsAdmin,newUserCreationRequest, HttpStatus.CREATED);
-        assertThat((String)newUserResponse.get("userIdentifier")).isNotEmpty();
+        newUserCreationRequest.setJurisdictions(new ArrayList<>());
+        Map<String, Object> newUserResponse = professionalApiClient.addNewUserToAnOrganisation(orgIdentifierResponse, hmctsAdmin, newUserCreationRequest, HttpStatus.BAD_REQUEST);

```

Comment:
please append the meaning-full description in the test case .
Example : add_new_user_to_organisation_with_no_jurisdiction_provided_should_return_400

Reply:
Done